cl/citations/fixtures/opinions_matching_citations.json

```diff
@@ -379,7 +379,7 @@
             "date_modified": "2015-08-15T14:10:56.801Z",
             "extracted_by_ocr": false,
             "author": 2,
-            "plain_text": "Blah blah Foo v. Bar 1 U.S. 1, 77 blah blah. Asdf asdf Qwerty v. Uiop 2 F.3d 2, 555. Also check out Foo, 1 U.S. 1, at 99. Then let's cite Qwerty, supra, at 666. See also Foo, supra, at 101 as well. Another full citation is Lorem v. Ipsum 1 U. S. 50. Quoting Qwerty, “something something”, 2 F.3d 2, at 559. This case is similar to Fake, supra, and Qwerty supra, as well. This should resolve to the foregoing. Ibid. This should also convert appropriately, see Id., at 577. This should fail to resolve because the reporter and citation is ambiguous, 1 U. S., at 51. However, this should succeed, Lorem, 1 U.S., at 52.",
+            "plain_text": "Blah blah Foo v. Bar 1 U.S. 1, 77 blah blah. Asdf asdf Qwerty v. Uiop 2 F.3d 2, 555. Also check out Foo, 1 U.S. at 99. Then let's cite Qwerty, supra, at 666. See also Foo, supra, at 101 as well. Another full citation is Lorem v. Ipsum 1 U. S. 50. Quoting Qwerty, “something something”, 2 F.3d 2, at 559. This case is similar to Fake, supra, and Qwerty supra, as well. This should resolve to the foregoing. Ibid. This should also convert appropriately, see Id., at 577. This should fail to resolve because the reporter and citation is ambiguous, 1 U. S., at 51. However, this should succeed, Lorem, 1 U.S., at 52.",
```

Comment:
This change is not necessary to make the tests pass; it was just a correction to the existing test case because I had previously said that there were a certain number of full case citations and a certain number of short case citations in this test string. Because both get counted, the test does not not hinge on which is which, but I just wanted to be consistent with the documentation.